hathor/sysctl/runner.py

```diff
@@ -75,6 +82,10 @@ def deserialize(self, value_str: str) -> Any:
         """Deserialize a value sent by the client."""
         if len(value_str) == 0:
             return ()
+
+        if re.match(array_pattern, value_str):
```

Comment:
This change seems unrelated to this PR. Is it a bug fix? If yes, it's missing in the acceptance criteria. Anyway, I think the fix might be wrong.

Reply:
Why?

Reply:
But yes, this is a fix for array parsing, and is being tested with [this line](https://github.com/HathorNetwork/hathor-core/pull/714/files#diff-6075f7992b86c5c2974496f5bf63e7f81b8286c675967d8c84c47efff0f690e2R104). Without this we can't deserialize an array with more than 1 element, because it is interpreted as a tuple.

Reply:
I updated the acceptance criteria.